projects/packages/forms/src/blocks/field-slider/edit.js

```diff
@@ -0,0 +1,133 @@
+import { useBlockProps, useInnerBlocksProps, InspectorControls } from '@wordpress/block-editor';
+import {
+	__experimentalHStack as HStack, // eslint-disable-line @wordpress/no-unsafe-wp-apis
+	__experimentalNumberControl as NumberControl, // eslint-disable-line @wordpress/no-unsafe-wp-apis
+	PanelBody,
+	RangeControl,
+} from '@wordpress/components';
+import { useEffect, useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import JetpackFieldControls from '../shared/components/jetpack-field-controls';
+
+export default function SliderFieldEdit( props ) {
```

Comment:
```suggestion
export default function SliderFieldEdit( props ) {
	useFormWrapper( props );
```
Adding that line will force a form whenever the slider is directly inserted on a post